File: javascript/2166.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift();};
const repetitions = parseInt(prompt());
var fraction = 0;
var SR2;


for (let i = 0; i < repetitions; i++) {
  fraction = 1 / (2 + fraction);
}

SR2 = 1 + fraction;
console.log(SR2.toFixed(10));

File: java/2166.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int repetitions = sc.nextInt();
        
        double fraction = 0.0;
        double SR2;
        
        for (int i = 0; i < repetitions; i++) {
            fraction = 1.0 / (2.0 + fraction);
        }
        
        SR2 = 1.0 + fraction;
        
        System.out.printf("%.10f%n", SR2);
        
        sc.close();
    }
}
